harness/types/capabilities.ts

```diff
@@ -58,6 +59,7 @@ let sdkCapabilities: { [key: string]: string[] } = {
         Capabilities.lastModifiedHeader,
         Capabilities.sdkConfigEvent,
         Capabilities.clientUUID,
+        //Capabilities.v2Config,
```

Comment:
should this be commented out?

Reply:
yeah or else it will fail since go on main does not have the code yet

Reply:
can probably leave it out, or add a todo